autogen.sh

```diff
@@ -163,6 +163,12 @@ fi
 
 conf_flags="--enable-maintainer-mode --enable-compile-warnings" #--enable-iso-c
 
+echo "MONO_CONC_GC is $MONO_CONC_GC"
+
+if test x$MONO_CONC_GC != x; then
+conf_flags="$conf_flags --with-sgen-default-concurrent"
+fi
+
```

Comment:
I don't think we should add an env variable for one of the mirriad configure options we have.